src/push-state-tree.js

```diff
@@ -106,49 +156,63 @@ var elementPrototype = typeof HTMLElement !== 'undefined' ? HTMLElement : Elemen
 
 function PushStateTree(options) {
   options = options || {};
-  options[USE_PUSH_STATE] = options[USE_PUSH_STATE] !== false;
 
   // Force the instance to always return a HTMLElement
   if (!(this instanceof elementPrototype)) {
     return PushStateTree.apply(PushStateTree.createElement('pushstatetree-route'), arguments);
   }
 
-  this.VERSION = VERSION;
+  this.eventStack = {
+    leave: [],
+    change: [],
+    enter: [],
+    match: []
+  };
 
-  // Setup options
-  for (var prop in options) {
-    if (options.hasOwnProperty(prop)) {
-      this[prop] = options[prop];
+  //TODO: emcapsulate this
```

Comment:
actually it's already because of the `let` there the comment became deprecated, thanks for the typo fixing :)